inlong-manager/manager-client-tools/src/main/java/org/apache/inlong/manager/client/cli/CommandCreate.java

```diff
@@ -71,7 +71,7 @@ void run() {
                 }
             } catch (Exception e) {
                 System.out.println("Create group failed!");
-                System.out.println(e.getMessage());
```

Comment:
Just printing the message of the exception is enough, thanks.

Reply:
The main reason is that the e.getMessage() method cannot see the abnormal link, and the abnormal information is not enough.